packages/ui5-application-writer/templates/README.md

```diff
@@ -0,0 +1,40 @@
+## Application Details
```

Comment:
This file should be in `./templates/core`

Reply:
great I will move, thanks